### ISSUE TITLE
Automatic mixed precision design spec

### ISSUE THREAD

**Issue by nobradovictt** [tenstorrent/tt-mlir#575]
Try lowering data format of specific ops. Investigate what can be salvaged from Pybuda AMP implementation.

**Comment by staylorTT** [comment]
@vcanicTT Did we end up with a MP design spec we can reference here?

**Comment by vcanicTT** [comment]
@staylorTT Yes, the initial version is finished. I'll read the document again, because it was long time ago, so I'll publish it.
